src/Microsoft.Azure.SignalR/Internal/AzureSignalRMarkerService.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class AzureSignalRMarkerService
+    {
+        internal bool UseAzureSignalRFlag { set; get; }
```

Comment:
naming: `UseAzureSignalR` is enough?

Reply:
Or `IsUseAzureSignalR`? Do we have some naming convention of adding `Is` prefix in bool property

Reply:
How about `IsConfigured` ?